docs/troubleshooting.md

```diff
@@ -276,4 +276,30 @@ The solution is to include all files that are mutated in your project's tsconfig
 }
 ```
 
-This shouldn't change anything about your Angular project. Your just being a bit more explicit in which files you want to include.
+This shouldn't change anything about your Angular project. You're just being a bit more explicit in which files you want to include.
+
+### Testing takes very long when using pnpm as package manager
+
+**Symptom**
+
+Running mutation tests takes longer than expected when using pnpm in combination with TypeScript.
```

Comment:
Thank you, I've updated the docs. I still mention typescript (in the form of typescript-checker) because this should be the only plugin causing trouble for newer projects. The others should be added by ```stryker init```.  If you have any more feedback I'll be happy to process it during the hackathon tomorrow! 🤠